### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -20645,11 +20711,15 @@ somethign->something
 somethimes->sometimes
 somethimg->something
 somethiong->something
+sometiem->sometimes
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Should be `sometime`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -20645,11 +20711,15 @@ somethign->something
...
 sometiems->sometimes
+sometihing->something
 sometihng->something
+sometiles->sometimes
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Or `some tiles`

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (pabs3) [reply codespell_lib/data/dictionary.txt]: In the location where I saw this typo, `sometimes` was clearly the right correction. Also, I feel like in code and reading some example sentences for `sometime`, the `sometimes` correction is mostly going to be the correct one. I will add sometime as a secondary correction though.

https://codesearch.debian.net/search?q=%5Cbsometiem%5Cb&literal=0
https://www.merriam-webster.com/dictionary/sometime

**Comment by pabs3** [comment]
Fixed the requested changes.

Sorry about the delay, it seems my email drops things from github sometimes.

**Comment by larsoner** [comment]
Thanks @pabs3

**Pull request merged**